### ISSUE TITLE
cannot open workbook by file name. 

### ISSUE THREAD

**Issue by doraemon1293** [ZoomerAnalytics/xlwings#804]
My code:
```
import xlwings as xw

app = xw.App()
fn = r"abc.xlsx"
dataWb = app.books.open(fn)
```

Error
```
Traceback (most recent call last):
  File "C:\project\learnXlwings\t.py", line 7, in <module>
    dataWb = app.books.open(fn)
  File "C:\Python35\lib\site-packages\xlwings-dev-py3.5.egg\xlwings\main.py", line 2746, in open
    throw = not os.path.samefile(impl.fullname, fullname)
  File "C:\Python35\lib\site-packages\xlwings-dev-py3.5.egg\xlwings\_xlwindows.py", line 488, in fullname
    return self.xl.FullName
AttributeError: 'NoneType' object has no attribute 'FullName'
```

In _xlwindows.py Class Book

```
    @property
    def fullname(self):
        return self.xl.FullName
```

self.xl equals to None

Python 3.5
win7-64
xlwings 0.11.6

**Comment by fzumstein** [comment]
Your code is correct. Does Excel open when you do `xw.App()` and does it have an empty workbook?

**Comment by doraemon1293** [comment]
Thank for your reply.

Yes it does.

Today I commented line 2742--2753 in main.py  and decrease the indentation of 2754, then everything works fine.

But I don't know whether it has any side effect.

**Comment by Razmaniak** [comment]
I have the same issue with latest xlwings package. Unable to connect to existing workbook by name through the book or books command.
app
    wb=app.books.open(dest_file)
  File "C:\Python36-32\lib\site-packages\xlwings\main.py", line 2746, in open
    throw = not os.path.samefile(impl.fullname, fullname)
  File "C:\Python36-32\lib\site-packages\xlwings\_xlwindows.py", line 488, in fullname
    return self.xl.FullName
AttributeError: 'NoneType' object has no attribute 'FullName'
 Excel is closed or opened does not makes sense... 
Windows 10 - 64 bit
Python 3.6.4 - 32 bit
xlwings 0.11.6

**Comment by fzumstein** [comment]
When do downgrade to 0.11.5, does it work properly?

**Comment by Razmaniak** [comment]
I've not try it... I've try and response you... Thank you for answer!

**Comment by Razmaniak** [comment]
Unfortunately I could not find xlwings version 0.11.5... Could you please give me link for this package?

**Comment by Razmaniak** [comment]
I've found 0.11.5 version on following destination https://testpypi.python.org/pypi/xlwings. Now it seems that it works without problems. Thanks fr rapid reply!

**Comment by fzumstein** [comment]
can you check if it fails again when you upgrade to 0.11.6 again?

**Comment by Razmaniak** [comment]
Unfortunately no success!!!((( The same error after upgrade to version 0.11.6. I've rollback to version 0.11.5. Could I write some questions regarding api directly on your email? Thank you!

**Comment by cloud1412a** [comment]
I found the same problem with 0.11.6.  Fixed it after rollback to 0.11.5.

**Comment by ja14000** [comment]
Same issue, rolling back to 0.11.5 fixes it for me too.

**Comment by fzumstein** [comment]
I'll look into it today.

**Comment by fzumstein** [comment]
I've just released 0.11.7 that should fix this issue.
